### PR TITLE
OSF 2141

### PR DESCRIPTION
**Pull request by MerlinZhang**
Fixed bug 2141
Modified profile-page.css
Names spanning multiple lines are no longer cut off. The name, however, is now shifted up slightly to be aligned to the top of the profile image rather than middle - I've spoken to Caner, and this new alignment is a trade-off for preventing the name from being cut off.

### CODE REVIEW

File: website/static/css/pages/profile-page.css [hunk 1]
```diff
@@ -4,22 +4,19 @@
...
-    position: relative;
     bottom: 0;
 }
 
 span#profileFullname{
-    position: absolute;
-    bottom: 0;
-    margin-bottom: 20px;
+    display:inline;
+    margin-left: -2px;
```
Comment (caneruguz) [inline website/static/css/pages/profile-page.css]: It's not ideal to correct margins with - sizes. Check what other css is effecting margin here, possibly remove m-l-sm. 
See style guide for margin and padding helpers https://centerforopenscience.github.io/osf-style/style.html#layout-helper

File: website/static/css/pages/profile-page.css [hunk 2]
```diff
@@ -4,22 +4,19 @@
...
+    float:right;
+    top: 0;
     right: 0;
 }
 
 div.profile-fullname{
     display: block;
-    position: relative;
     bottom: 0;
 }
```
Comment (caneruguz) [inline website/static/css/pages/profile-page.css]: Check whether these rules are required now that layout positioning is changed.

File: website/static/css/pages/profile-page.css [hunk 3]
```diff
@@ -36,12 +33,12 @@ table.table.table-plain th, table.table.table-plain td {
 /* Profile gravatar
 -------------------------------------------------- */
 .profile-gravatar {
+    float: left;
     width:70px;
     height:70px;
+    margin-right: .3cm;
 }
```
Comment (caneruguz) [inline website/static/css/pages/profile-page.css]: That's interesting, why not another relative or absolute unit?

**Comment by HarryRybacki** [comment]
@MerlinZhang -- Please review and revise based on @caneruguz's comments

**Comment by HarryRybacki** [comment]
CR 1 :+1: !

**Pull request merged**